File: pages/mogura.js
$(function(){
  // var ws = new WebSocket("ws://192.168.1.224:8765/");
  var ws = new WebSocket("ws://localhost:8765");

  // サーバーからのメッセージ受信時のイベント
  ws.onmessage = function(message){

    data = JSON.parse(message["data"]);
    console.log(data);
  }

  $("#send_num").on("click", function(){
    const num = $("#num").val()
    console.log(num);
    ws.send(JSON.stringify({"type":"manager", "data":num}));

    // 送ったら隠す
    $("#num").hide()
    $("#send_num").hide()
  });
})

File: mogura.py
import asyncio
import websockets
import json

async def mogura():
    #async with websockets.connect("ws://192.168.1.224:8765") as websocket:
    #async with websockets.connect("ws://localhost:8765", ping_interval=None) as websocket:
    async with websockets.connect("ws://10.0.1.3:8765", ping_interval=None) as websocket:

        await websocket.send(json.dumps({"type": "mogura", "data": 1}))
        message = await websocket.recv()
        print(message)

        while 1:
            score = int(input("score:"))
            await websocket.send(json.dumps({"type": "score", "data":score}))
            message = await websocket.recv()
            print(message)


asyncio.run(mogura())
